tools/torchci/clickhouse_query_perf.py

```diff
@@ -129,7 +133,7 @@ def perf_compare(args: argparse.Namespace) -> None:
     # Split up the query execution and the stats collection because the stats
     # table needs time to populate. Also sleep for 10 seconds to the table more
     # time to populate
-    time.sleep(20)
+    time.sleep(30)
```

Comment:
Now that you need to change the value here, it seems useful to have this as a configurable parameter default to 30 seconds